rpcs3/Emu/System.cpp

```diff
@@ -439,6 +443,20 @@ void Emulator::Load(bool add_only)
 		}
 
 		LOG_NOTICE(LOADER, "Used configuration:\n%s\n", g_cfg.to_string());
+		
+		// Set RTM usage
+		g_use_rtm = utils::has_rtm() && (utils::is_haswell_or_broadwell() == utils::ModelNotHaswellBroadwell || g_cfg.core.enable_TSX);
+		if (utils::has_rtm())
+		{
+			if (g_use_rtm)
+			{
+				LOG_NOTICE(GENERAL, "TSX enabled");
+			}
+			else
+			{
+				LOG_NOTICE(GENERAL, "TSX disabled");
```

Comment:
Same goes for this

Reply:
![screen tsx](https://user-images.githubusercontent.com/38664966/39836500-e929b850-53d3-11e8-9188-8330bb090ca4.PNG)

Reply:
Was writing the same above: For Haswell/Broadwell Users the "| TSX" will appear if the CPU has the capabilities, regardless if it's enabled or not. Therefore this Info log will only show for them.

Reply:
If you insist on adding this, you should at least use a LOG_WARNING instead of a LOG_NOTICE
:)